source/schema.txt

```diff
@@ -335,6 +335,28 @@ To delete a location filter from the map:
 
 .. |query-build-type| replace:: :ref:`build-query`
 
+Troubleshooting
+---------------
+
+If the analysis of your schema times out, it might be because the 
+collection you are analyzing is very large and the operation is stopped 
+before the analysis is complete. Increase the value of ``maxTimeMS`` 
+to allow the operation time to complete.
+
+To increase the value of ``maxTimeMS``:
+
+1. In the query bar, expand :guilabel:`Options`.
+
+   .. figure:: /images/compass/schema-query-options.png
+      :alt: The Options button is on the right side of the query bar, 
+            next to the green Analyze button.
```

Comment:
I think we tend to `guilabel` button text in DevEd -- if you do the same, you might want to make this change!

```suggestion
            next to the green :guilabel:`Analyze` button.
```

Reply:
Normally we would but this is an extension of that alt text (the line wrapping is out of whack) and I don't think directives work there